fastestimator/dataset/data/breast_cancer.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Tuple
+
+import numpy as np
+from sklearn.datasets import load_breast_cancer
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+
+from fastestimator.dataset.numpy_dataset import NumpyDataset
+
+
+def load_data() -> Tuple[NumpyDataset, NumpyDataset]:
+    (X, y) = load_breast_cancer(True)
+    x_train, x_eval, y_train, y_eval = train_test_split(X, y, test_size=0.2)
```

Comment:
should we seed this split so that users always get exactly the same train and eval sets?

Reply:
I guess not, random splitting has been a convention for long.

Reply:
but here we're telling the end user 'this is your training data'. if they want a random split they can just use our .split() syntax later. I would be surprised to find that my training set and eval set were getting mixed up every time a ran an experiment

Reply:
I see your point.